Scripts/configure-ODBC-DSN.ps1

```diff
@@ -5,31 +5,52 @@
 
 $DSNNames = (Get-OdbcDsn).Name
 
-# AZURESQL
 
-Write-Host("Updating Server name and database name for AZURESQL")
+# AZURESQL
 
 if($DSNNames -contains "AZURESQL" -and (Get-OdbcDsn -Name "AZURESQL").Platform -eq "32-bit"){
+
+    Write-Host("Updating Server name and database name for AZURESQL")
+
     $sqlserver = "db-regression-" + $lansaVersion + ".database.windows.net" 
     Set-OdbcDsn -Name "AZURESQL" -DsnType "System" -Platform "32-bit" -SetPropertyValue @("Server=$sqlserver", "Database=$lansaVersion")
+
 }
 
+
 # Oracle
 
-Write-Host("Updating Server name for Oracle")
+if($DSNNames -contains "ora19cdb" -and (Get-OdbcDsn -Name "ora19cdb").Platform -eq "32-bit"){
+
+    $TNSNamesPath = "C:\oracle\instantclient_21_6\network\admin\tnsnames.ora"
+    $Content = Get-Content $TNSNamesPath
+    $oraServer = "ora" + $lansaVersion + ".cnyed5gpqwey.us-east-1.rds.amazonaws.com"
```

Comment:
You need to derive the endpoint from AWS.

Reply:
Not required anymore.